### ISSUE TITLE
"Add Package" button is not appearing as "Disabled" button in both modes of high contrast.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#360]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode before launching the application.

1. Launch Xamarin Workbooks application and create a new workbook.
2. Navigate to File -> Add Package. Invoke "Add Package" . It will open "Package Manager" window.
3. Observe the "Add Package" button without searching any packages in both modes of high contrast.

### Expected Behavior
"Add Package" should be displayed as disable button.
### Actual Behavior
"Add Package" button is hardly visible and is not following disabled button standards.
### User Impact
Low vision will not know whether it is disabled or not as per High Contrast standards.
### Screenshots
![14 hcb](https://user-images.githubusercontent.com/22956130/40724764-76e20d98-643f-11e8-9f82-c1711e8b0e6b.png)
![14 hcw](https://user-images.githubusercontent.com/22956130/40724765-77238322-643f-11e8-993d-cbf23422e8ec.png)
![14 normal](https://user-images.githubusercontent.com/22956130/40724767-77665fe4-643f-11e8-824b-5dbd8bb1c6dc.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1
